src/utils/requests.ts

```diff
@@ -1,13 +1,28 @@
 import { Request } from 'express'
+import { User } from '@openforis/arena-core'
+
+type ArenaRequest = { user: User } & Request
 
 const getHost = (request: Request): string | undefined => request.header('host')
+const getParams = (req: Request): Record<string, any> => {
+  const params = { ...req.query, ...req.params, ...req.body }
+  return Object.entries(params).reduce<Record<string, any>>((paramsAcc, [key, value]) => {
+    // Convert String boolean values to Boolean type
+    paramsAcc[key] = value === 'true' || value === 'false' ? value === 'true' : value
+    return paramsAcc
+  }, {})
+}
+
 const getServerUrl = (request: Request): string => `${request.protocol}://${request.get('host')}`
 const getUrl = (request: Request): string => request.url
+const getUser = (request: Request): User => (request as ArenaRequest).user
```

Comment:
Why is `request: Request` recast to `ArenaRequest` and not directly used as `ArenaRequest`?